src/views/Timeline.vue

```diff
@@ -54,43 +66,58 @@ export default {
       loading: false,
       loaded: false,
       proposals: {},
-      scope: this.$route.params.scope
+      scope: this.$route.params.scope,
+      state: 'all'
     };
   },
-  async created() {
-    this.loading = true;
-    const spaces =
-      this.scope === 'all' ? [] : Object.keys(this.favoriteSpaces.favorites);
-    try {
-      const proposals = await subgraphRequest(
-        `${process.env.VUE_APP_HUB_URL}/graphql`,
-        {
-          timeline: {
-            __args: {
-              spaces
-            },
-            id: true,
-            name: true,
-            start: true,
-            end: true,
-            state: true,
-            author: {
-              address: true,
-              name: true,
-              ens: true
-            },
-            space: {
+  watch: {
+    state() {
+      this.loadProposals();
```

Comment:
There should be the loading placeholder when we reload the page with another state